ui/imports/shared/stores/PermissionsStore.qml

```diff
@@ -97,7 +97,9 @@ QtObject {
         id: becomeMemberPermissionsModel
         sourceModel: root.permissionsModel
         function filterPredicate(modelData) {
-            return (modelData.permissionType == Constants.permissionType.member) &&
+            return (modelData.permissionType === Constants.permissionType.member ||
+                    modelData.permissionType === Constants.permissionType.admin ||
```

Comment:
Why the `admin` role is added here if the model is about becoming member / joining community??